x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/endpoint_details_content.tsx

```diff
@@ -70,83 +74,98 @@ export const EndpointDetailsContent = memo(
     const detailsResults = useMemo(() => {
       return [
         {
-          title: i18n.translate('xpack.securitySolution.endpoint.details.os', {
-            defaultMessage: 'OS',
-          }),
-          description: <EuiText>{details.host.os.full}</EuiText>,
+          title: (
+            <ColumnTitle>
+              <FormattedMessage
+                id="xpack.securitySolution.endpoint.details.os"
+                defaultMessage="OS"
+              />
+            </ColumnTitle>
+          ),
+          description: <EuiText size="xs">{details.host.os.full}</EuiText>,
         },
         {
-          title: i18n.translate('xpack.securitySolution.endpoint.details.agentStatus', {
-            defaultMessage: 'Agent Status',
-          }),
+          title: (
+            <ColumnTitle>
+              <FormattedMessage
+                id="xpack.securitySolution.endpoint.details.agentStatus"
+                defaultMessage="Agent Status"
+              />
+            </ColumnTitle>
+          ),
           description: <EndpointAgentStatus hostStatus={hostStatus} endpointMetadata={details} />,
         },
         {
-          title: i18n.translate('xpack.securitySolution.endpoint.details.lastSeen', {
-            defaultMessage: 'Last Seen',
-          }),
+          title: (
+            <ColumnTitle>
+              <FormattedMessage
+                id="xpack.securitySolution.endpoint.details.lastSeen"
+                defaultMessage="Last Seen"
+              />
+            </ColumnTitle>
+          ),
           description: (
-            <EuiText>
+            <EuiText size="xs">
               {' '}
-              <FormattedDateAndTime date={new Date(details['@timestamp'])} />
+              <FormattedDate value={details['@timestamp']} fieldName="" />
             </EuiText>
           ),
         },
         {
-          title: i18n.translate('xpack.securitySolution.endpoint.details.policy', {
-            defaultMessage: 'Policy',
-          }),
+          title: (
+            <ColumnTitle>
+              <FormattedMessage
+                id="xpack.securitySolution.endpoint.details.policy"
+                defaultMessage="Policy"
+              />
+            </ColumnTitle>
+          ),
           description: (
-            <EuiFlexGroup alignItems="center">
-              <EuiFlexItem grow={false}>
-                <EuiText>
-                  <EndpointPolicyLink
-                    policyId={details.Endpoint.policy.applied.id}
-                    data-test-subj="policyDetailsValue"
-                  >
-                    {details.Endpoint.policy.applied.name}
-                  </EndpointPolicyLink>
+            <EuiText size="xs" className={'eui-textBreakWord'}>
+              <EndpointPolicyLink
+                policyId={details.Endpoint.policy.applied.id}
+                data-test-subj="policyDetailsValue"
+                style={{ paddingRight: '5px' }}
```

Comment:
Consider using a static object for the `style` value or memoizing it in order to avoid unnecessary re-renders. Same below

Reply:
There's no need for memo here. this component is literally a link, the win on using memo is probably worse in performance than not using it.

I'll use className in this one as David suggested.